.github/workflows/observability-ci.yml

```
@@ -77,39 +77,39 @@ jobs:
     outputs:
       dashboard-count: ${{ steps.count.outputs.count }}
       validation-status: ${{ steps.validate.outputs.status }}
-    
+
     steps:
       - name: Checkout code
         uses: actions/checkout@v4
-      
+
       - name: Set up Python
         uses: actions/setup-python@v4
```

actionlint (1.7.10)

[error] the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)